pkg/fsutil/fsutil_unix.go

```diff
@@ -0,0 +1,11 @@
+//go:build !windows && !plan9
```

Comment:
This file is a platform-specific part of search.go, so better name it `search_unix.go`.